libcipher/utils.py

```diff
@@ -0,0 +1,62 @@
+import io
+import sys
+import string
+
+
+class EnglishChecker():
+    """This class serves the purpose of checking if a statement is
+       English or not. Two filter methods accomplish this by ensuring:
+        1. At least 85% of the message comprises of english alphabets
+        2. At least 25% of the words in the sentence are contained in
+           the dictionary file that the user provides
+    """
+
+    def __init__(self, filename):
+        self.filename = filename
```

Comment:
Just call the argument `dict_file` or `dictionary_file` instead of `filename`. Since it is _not_ a filename, but a file object. Same with the attribute as well.